Sources/StreamChat/Models/BlockedUser.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A type representing a blocked user. `BlockedUser` is an immutable snapshot of a blocked user entity at the given time.
+///
+public class BlockedUser {
```

Comment:
It is better that our models are value types 
```suggestion
public struct BlockedUser {
```

Reply:
Updated 👍